website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -56,8 +56,8 @@ Notifications are sent as the run progresses, and can be triggered on one or mor
 | Applying           | `"run:applying"`        | A run enters the [Apply stage](/cloud-docs/run/states#8-the-apply-stage), where Terraform makes the infrastructure changes described in the plan.                            |
 | Completed          | `"run:completed"`       | A run has completed successfully.                                     |
 | Errored            | `"run:errored"`         | A run has terminated early due to error or cancellation.                                          |
-| Drifted            | `"assessment:drifted"`  | Terraform Cloud detected configuration drift. This is only available if [drift detection](/cloud-docs/workspaces/settings/drift-detection) is enabled for the workspace.                                                     |
-| Drift Check Failed | `"assessment:failed`"   | A drift detection assessment failed. This is only available if [drift detection](https://www.terraform.io/cloud-docs/workspaces/settings/drift-detection) is enabled for the workspace.                                          |
+| Drifted            | `"assessment:drifted"`  | Terraform Cloud detected configuration drift. This is only available if drift detection is enabled for the workspace.  Not available in Terraform Enterprise.                                                   |
+| Drift Check Failed | `"assessment:failed`"   | A drift detection assessment failed. This is only available if drift detection is enabled for the workspace.   Not available in Terraform Enterprise.                                       |
```

Comment:
```suggestion
| Drifted            | `"assessment:drifted"`  | Terraform Cloud detected configuration drift. This option is only available if drift detection is enabled for the workspace.  Not available in Terraform Enterprise.                                                   |
| Drift Check Failed | `"assessment:failed`"   | A drift detection assessment failed. This option is only available if you enable drift detection for the workspace.  Not available in Terraform Enterprise.                                       |
```

Suggest changing "This is only available if drift detection is enabled" to "This option is only available if you enable drift detection" to avoid passive voice and "This" with no subject.